test/event_test.rb

```diff
@@ -135,25 +135,32 @@ def test_attendees_returns_an_array_of_attendees_with_assigned_numbers
   def test_updates_event_with_params
     VCR.use_cassette 'event-update' do
       find_event
-      response = @event.update name: 'Movember 2014'
-      assert response, 'Should return true after updating'
+      event = @event.update 'event.name.html' => 'Movember 2014'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.